Sources/SwifterSwift/Dispatch/DispatchQueueExtensions.swift

```diff
@@ -41,6 +41,19 @@ public extension DispatchQueue {
 
         return DispatchQueue.getSpecific(key: key) != nil
     }
+    
+    /// SwifterSwift: Runs passed closure asynchronous after certain time interval
+    ///
+    /// - Parameter delay: The time inverval after which the closure will run.
+    /// - Parameter qos: Quality of service at which the work item should be executed.
+    /// - Parameter flags: Flags that control the execution environment of the work item.
+    /// - Parameter work: The closure to run after certain time interval.
+    func asyncAfter(delay: Double,
+                    qos: DispatchQoS = .unspecified,
+                    flags: DispatchWorkItemFlags = [],
+                    execute work: @escaping @convention(block) () -> Void) {
```

Comment:
I think we don't need to explicitly require a block convention here, we can have a swift closure here. As far as I know, swift to block are implicit convertible...